Formula/socket_vmnet.rb

```diff
@@ -18,7 +18,7 @@ class SocketVmnet < Formula
 
   keg_only "#{HOMEBREW_PREFIX}/bin is often writable by a non-admin user"
 
-  depends_on :macos
```

Comment:
Still need this to make formulae macOS-only. Other DSL only lists minimum macOS but allows Linux.

Reply:
➕